dist.sh

```diff
@@ -70,16 +78,27 @@ for platform in $platforms; do
 
     cp -r $template_dir/* $dist_path/
 
-    cp $core_dir/build/libs/*.jar $dist_path/workcraft.jar
+    # resource_dir is simply the distribution root. OS X needs special care.
+    resource_dir=$dist_path
```

Comment:
If `resource_dir` is simply the distribution root, it seems to me like the name isn't appropriate. I would get confused as to why we're copying jars into a resource dir.